### PR TITLE
Alan realisation registerer

### PR DESCRIPTION
**Pull request by kokoal**

### CODE REVIEW

File: src/AppBundle/Controller/RealisationController.php [hunk 1]
```diff
@@ -4,14 +4,44 @@
...
     public function listAction(Request $request)
     {
     }
 
-    public function registerAction(Request $request)
+    public function registerAction(Request $request, string $campaignId)
     {
+        $form = $this->createForm(RealisationRegistrationType::class, new RealisationRegistration());
+
+        if ($request->isMethod('post')) {
```
Comment (roincent) [inline src/AppBundle/Controller/RealisationController.php]: Remove if statement -> see best practice for forms => http://symfony.com/doc/current/best_practices/forms.html

File: src/AppBundle/Controller/RealisationController.php [hunk 2]
```diff
@@ -4,14 +4,44 @@
...
+
+                return $this->redirect("/");
+            }
+        }
+
+        $campaign = $this->get('app.campaign.repository')->findOneById($campaignId);
+        return $this->render(
+            'AppBundle:Realisation:realisationRegistration.html.twig', [
+                'realisationRegistrationForm' => $form->createView(),
+                'campaign' => $campaign
```
Comment (roincent) [inline src/AppBundle/Controller/RealisationController.php:41]: What about a not found campain? -> Need to be handle!

File: src/AppBundle/Services/RealisationRegisterer.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+    public function create(RealisationRegistration $realisationRegistration, $campaignId)
+    {
+        $campaign = $this->campaingRepository->findOneById($campaignId);
+        if ($campaign === null) {
+            throw new Exception('PAS DE CAMPAGNE');
+        }
+
+        $file = $realisationRegistration->file;
+        $fileName = sprintf('%s.%s', md5(uniqid()), $file->guessExtension());
```
Comment (roincent) [inline src/AppBundle/Services/RealisationRegisterer.php:30]: There will be a service for uuid in the next commit :)

**Review by roincent — CHANGES_REQUESTED** [review]

File: src/AppBundle/Controller/RealisationController.php (hunk 2 above)
Comment (stacyharper) [reply src/AppBundle/Controller/RealisationController.php:41]: Handeled in another commit ;) But in wrong branch sry..

**Pull request merged**